xml/sec_images_hardened.xml

```diff
@@ -26,12 +26,44 @@
  </para>
  <para>
   All images are pre-hardened to the extent they can safely be hardened without
-  causing problems in public cloud frameworks. After instance creation, users
+  causing problems in public cloud frameworks. Certain rules can only be
+  applied after instance creation, for example:
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    Rules that require having passwords set up. Passwords are not yet configured
+    when images are built.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Rules that affect the network configuration. Networking is set up during
+    instance creation, therefor it is not yet possible to limit access.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Rules for custom partitioning. &suse;'s public cloud images use a standard
+    partitioning with a single root file system. You can create additional
+    partitions after instance creation.
```

Comment:
Looks good over all, but remove "You can create additional
    partitions after instance creation." We really don't want to point them down that road. I have heard not not verified that having separate /usr, /var,/tmp etc can cause upgrade problems with cloud images,